saleor/graphql/giftcard/tests/mutations/test_gift_card_bulk_deactivate.py

```diff
@@ -0,0 +1,134 @@
+import graphene
+
+from .....giftcard import GiftCardEvents
+from .....giftcard.models import GiftCard, GiftCardEvent
+from ....tests.utils import assert_no_permission, get_graphql_content
+
+MUTATION_GIFT_CARD_BULK_DEACTIVATE = """
+    mutation GiftCardBulkDeactivate($ids: [ID]!) {
+        giftCardBulkDeactivate(ids: $ids) {
+            count
+            errors {
+                code
+                field
+            }
+        }
+    }
+"""
+
+
+def test_gift_card_bulk_deactivate_by_staff(
```

Comment:
This test should also check that these gift cards are deactivated after this mutation.